gateway/src/apicast/cli/environment.lua

```diff
@@ -148,7 +148,7 @@ function _M:add(env)
 
     local config = loadfile(path, 't', {
         print = print, inspect = require('inspect'), context = self._context,
-        tonumber = tonumber, tostring = tostring,
+        tonumber = tonumber, tostring = tostring, os = { getenv = resty_env.value },
```

Comment:
I think that overriding `getenv` here can be a bit confusing.
Although I get that you're doing this to be able to use it in the environment files.

Reply:
Yep. I think it is more useful to expose `resty_env.value` than plain `resty_env.get` or `os.getenv`.
And the only difference is that it does not return empty strings. Which is actually useful for default values.